### PR TITLE
Adding template for Super Secure stop screen

### PR DESCRIPTION
**Pull request by PhilipGlover**
- adds template and routing for screen, but not the validation check that would navigate to it.
- if `http://chs.local/persons-with-significant-control-verification/stop/super-secure?companyNumber=00006400` is used the back link url will be set accordingly.  I am presuming the validation story later will pass the company Number to the stop screen handler.

IDVA3-2459

Screenshot 2024-12-03 at 1 45 50 PM

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/constants.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ export const servicePathPrefix = "/persons-with-significant-control-verification
 const urlWithTransactionIdAndSubmissionId = "/transaction/:transactionId/submission/:submissionId";
 
 export enum STOP_TYPE {
+    SUPER_SECURE = "super-secure",
```
Comment (hepsimo) [inline src/constants.ts:5]: Keep these alphabetical?

**Review by hepsimo — APPROVED** [review]
Changes look good, including tests

File: src/constants.ts (hunk 1 above)
Comment (PhilipGlover) [reply src/constants.ts]: okey doke.  donei

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**